### PR TITLE
ASAP-470 Add teams reference and display manuscript title in the frontend

### PR DESCRIPTION
**Pull request by gabiayako**
This PR

- Adds the team reference to the `manuscripts` content model
Screenshot 2024-05-17 at 12 18 03

- Displays the manuscript in the frontend
Screenshot 2024-05-17 at 12 17 04

### CODE REVIEW

File: packages/contentful/src/crn/queries/manuscript.queries.ts [hunk 1]
```diff
@@ -8,6 +8,13 @@ export const manuscriptContentQueryFragment = gql`
       id
     }
     title
+    teamsCollection(limit: 10) {
```
Comment (AkosuaA) [inline packages/contentful/src/crn/queries/manuscript.queries.ts:11]: Would we be having multiple teams assigned to a manuscript?

Comment (gabiayako) [reply packages/contentful/src/crn/queries/manuscript.queries.ts:11]: Yeah, we're going to have this field in the form in the future :)
Screenshot 2024-05-20 at 09 34 32

Comment (AkosuaA) [reply packages/contentful/src/crn/queries/manuscript.queries.ts:11]: Ah yes, I remember the discussion on this. Thanks! The teamId in the model confused me but it makes sense to add that when it becomes relevant. Looks good!

Comment (gabiayako) [reply packages/contentful/src/crn/queries/manuscript.queries.ts:11]: thanks 😄

**Pull request merged**